### ISSUE TITLE
Product object missing from the items Entity

### ISSUE THREAD

**Issue by theyough** [PaddleHQ/paddle-php-sdk#56]
### Describe the bug

from your documentation
https://developer.paddle.com/webhooks/subscriptions/subscription-created

Inside the array `items` you show the object `product` but inside the Class `SubscriptionItem` it's missing

### Steps to reproduce

see the code snippets

### Expected behavior

see the code snippets

### Code snippets

```PHP
namespace Paddle\SDK\Notifications\Entities\Subscription;

use Paddle\SDK\Notifications\Entities\DateTime;

class SubscriptionItem
{
    private function __construct(
        public SubscriptionItemStatus $status,
        public int $quantity,
        public bool $recurring,
        public \DateTimeInterface $createdAt,
        public \DateTimeInterface $updatedAt,
        public \DateTimeInterface|null $previouslyBilledAt,
        public \DateTimeInterface|null $nextBilledAt,
        public SubscriptionTimePeriod|null $trialDates,
        public SubscriptionPrice $price,
// Here, it's missing the Product object
    ) {
    }
...
```
```


### PHP version

PHP 8.3

### SDK version

paddlehq/paddle-php-sdk 1.1.0

### API version

Paddle Version 1

### Additional context

_No response_

**Comment by heymcgovern** [comment]
Hey @theyough, Michael from the DX team here 👋 

Thanks for the detailed bug report. You're absolutely right — the `product` object is missing from `SubscriptionItem`. [We added this to the API in July](https://developer.paddle.com/changelog/2024/subscription-items-product), but we haven't yet brought it to the PHP SDK. We've created a ticket to do this, and we’re aiming to tackle it within the next month.

We’ll keep you updated! 📫
